docs/jwst/flatfield/main.rst

```diff
@@ -68,3 +75,16 @@ extract a matching subarray from the reference file data arrays and apply them
 to the science data. Hence full-frame reference files can be
 used for both full-frame and subarray science exposures, or subarray-dependent
 reference files can be provided if desired.
+
+Error Propagation
+-----------------
+The VAR_POISSON and VAR_RNOISE variance arrays are divided by the square
+of the flat-field value for each pixel. A flat-field variance array,
+VAR_FLAT, is created from the science exposure and flat-field reference
+file data using the following formula:
+
+.. math::
+   VAR\_FLAT = SCI_{science}^{2} / SCI_{flat}^{2} * ERR_{flat}^{2}
+
+The total ERR array is updated as the quadratic sum of VAR_POISSON,
```

Comment:
I would interpret "quadratic sum" to mean just the sum of the squares.  The expression "add in quadrature" means the square root of the sum of the squares.